README.md

```diff
@@ -214,6 +215,15 @@ iterable([ 1, 2, 3, 4 ]); // 4, 16
 ```
 This is more memory efficient of using array methods as it doesn't require to build intermediate arrays.
 
+## cursor
+It returns all items with the previous elements returned. Inside an array of a determined size.
```

Comment:
I don't think it should start with undefined. At no point should reasonable code care what the negative second value is('nt).

Reply:
makes sense. FIxed